src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.html

```diff
@@ -76,7 +76,6 @@
                   class="kuiLink"
                   href="https://en.wikipedia.org/wiki/ISO_week_date"
                   target="_blank"
-                  title="Wikipedia: ISO Week Date"
```

Comment:
I noticed that the article is from January 2012, over 5 years ago. The main reason it suggests not using `title` is that not all browsers support it. I checked https://caniuse.com/#search=title and that seems to no longer be the case. Thoughts?